### DISCUSSION TITLE
Problem pairing LIDL/Livarno Lux switch and dimming light remote control

### DISCUSSION THREAD

**Discussion by sanderjohan** [Koenkk/zigbee2mqtt#5948] [Devices]
When I pair my Livarno remote (model HG06323) in Domoticz it is is paired as Model FB20-002 but I do not see any devices showing up. Furthermore I get the error.

2021-01-27 23:01:27.501 Python Plugin System: (Zigbee2MQTT) This plugin does not support zigbee device with model "FB20-002" yet
2021-01-27 23:01:27.501 Python Plugin System: (Zigbee2MQTT) If you would like plugin to support this device, please create ticket by this link: https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues/new?labels=new+device&template=new-device-support.md

I;am running v.1.17.0 (zStack3x0 20200925) and entry in database.db looks like.

{"id":17,"type":"EndDevice","ieeeAddr":"0xec1bbdfffeab6f9d","nwkAddr":63218,"manufId":4098,"manufName":"_TYZB01_bngwdjsr","powerSource":"Battery","modelId":"TS1001","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":260,"inClusterList":[0,1,3,4,4096],"outClusterList":[25,10,3,4,5,6,8,4096],"clusters":{"genBasic":{"attributes":{"modelId":"TS1001","manufacturerName":"_TYZB01_bngwdjsr","powerSource":3,"zclVersion":3,"appVersion":81,"stackVersion":0,"hwVersion":1,"dateCode":""}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":81,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1611784379831}

How pair the remote properly?

**Comment by jamesd256** [comment]
Hey OP, Do you know how to put the switch into pairing mode?

Thanks

**Reply by frontibit** [reply]
press the O and I button for 5-10 seconds
